### PR TITLE
Remove non-working samples and add a redirect sample

### PR DESCRIPTION
**Pull request by sameerag**
Updating the samples:

- removing unused samples under VanillaJSTestApp folder
- adding index_redirect.html for redirect usecases

### CODE REVIEW

File: lib/msal-core/samples/VanillaJSTestApp/index.html [hunk 1]
```diff
@@ -19,17 +19,12 @@
...
-            graphEndpoint: "https://graph.microsoft.com/v1.0/me",
-            loginType: 'POPUP'
-    };*/
 
-    var msalConfig = {
+    // initialize MSAL
+    const msalConfig = {
         auth: {
             clientId: "Enter_your_client_id",
+            // clientId: "245e9392-c666-4d51-8f8a-bfd9e55b2456",
```
Comment (DarylThayil) [inline lib/msal-core/samples/VanillaJSTestApp/index.html]: do we want to leave this in here?

Comment (jasonnutter) [reply lib/msal-core/samples/VanillaJSTestApp/index.html]: I think so, its helpful to have a demo client id available.

Comment (DarylThayil) [reply lib/msal-core/samples/VanillaJSTestApp/index.html]: Is it an internal only client id?

Comment (DarylThayil) [reply lib/msal-core/samples/VanillaJSTestApp/index.html]: like is there concern around exposing to others

Comment (jasonnutter) [reply lib/msal-core/samples/VanillaJSTestApp/index.html]: Nope, its been in the repo for a while, hasn't it? And its only setup for localhost.

Comment (sameerag) [reply lib/msal-core/samples/VanillaJSTestApp/index.html]: Yes. I created this application (in the portal) just for demo purposes, so no issues in making this public. I left it alone here in the sample to have it ready for anyone to test it.

**Pull request merged**